lib/core/src/lib/notifications/components/notification-history.component.scss

```diff
@@ -54,18 +31,67 @@ $notification-item-height: 72px;
         }
     }
 
-    &-notification-history-list #{$mat-subheader} {
-        display: flex;
-        justify-content: space-between;
-        align-items: center;
-    }
-
     &-notification-history-menu:has(.adf-notification-history-list) {
-        .adf-notification-history-menu-item {
-            cursor: pointer;
-            height: $notification-item-height;
+        .adf-notification-history-list-header {
+            padding: 10.5px 16px;
+            display: flex;
+            justify-content: space-between;
             align-items: center;
-            padding-left: 16px;
+        }
+
+        .adf-notification-history-menu-title {
+            line-height: 28px;
+        }
+
+        .adf-notification-history-mark-as-read {
+            display: flex;
+            padding: 10px;
+            width: auto;
+            margin: 4px 0;
+
+            &:hover,
+            &:focus,
+            &:active {
+                background: none;
+            }
+
+            #{$mat-icon} {
```

Comment:
Can we add a class to the mat-icon and use that instead? Since we want to minimise the usage of mat-selectors unless absolutely necessary?

Reply:
done, ty